neovim/plugin/script_host.py

```diff
@@ -65,7 +65,9 @@ def setup(self, nvim):
         sys.stderr = RedirectStream(lambda data: nvim.err_write(data))
         # Setup the DirChanged listener
         cid = nvim.channel_id
-        nvim.command('au DirChanged * call rpcrequest(%d, "python_chdir", v:event)' % cid)
+        cmd = 'au DirChanged * call rpcrequest(%d, "python_chdir", v:event)' \
```

Comment:
Please use `autocmd` instead of `au`, which makes it easier go grep for when searching for autocmd-related things.

Reply:
Sure.